script.js

```diff
@@ -0,0 +1,60 @@
+const CLAIM_BUTTON = document.getElementById("submitBtn");
+
+const INPUT_PASSWORD = document.getElementById("password");
+const INPUT_EMAIL = document.getElementById("email");
+const INPUT_LNAME = document.getElementById("lName");
+const INPUT_FNAME = document.getElementById("fName");
+
+const ERROR_PASSWORD = document.getElementById("passwordError");
+const ERROR_EMAIL = document.getElementById("emailError");
+const ERROR_LNAME = document.getElementById("lnameError");
+const ERROR_FNAME = document.getElementById("fnameError");
+
+const BORDER_PASSWORD = document.getElementById("passwordBorder");
+const BORDER_EMAIL = document.getElementById("emailBorder");
+const BORDER_LNAME = document.getElementById("lnameBorder");
+const BORDER_FNAME = document.getElementById("fnameBorder");
```

Comment:
Although you are selecting this for the purpose of changing the input border you are still selecting the input itself. So just for clarity I would name it appropriately.

```suggestion
const PASSWORD_INPUT = document.getElementById("passwordInput");
const EMAIL_INPUT = document.getElementById("emailInput");
const LNAME_INPUT = document.getElementById("lnameInput");
const FNAME _INPUT= document.getElementById("fnameInput");
```

**Note:** You will have to update your HTML as well